README.md

```diff
@@ -63,23 +67,24 @@ Kubernetes version numbers.
 
 #### Branches and tags.
 
-We will create a new branch and tag for each increment in the major version number or
-minor version number. We will create only a new tag for each increment in the patch
+We will create a new branch and tag for each increment in the minor version
+number. We will create only a new tag for each increment in the patch
 version number. See [semver](http://semver.org/) for definitions of major,
 minor, and patch.
 
 The master branch will track HEAD in the main Kubernetes repo and
-accumulate changes. Consider HEAD to have the version `x.(y+1).0-alpha` or
-`(x+1).0.0-alpha` (depending on whether it has accumulated a breaking change or
-not), where `x` and `y` are the current major and minor versions.
+accumulate changes. Consider HEAD to have the version `x.(y+1).0-alpha`,
+where `x` and `y` are the current major and minor versions.
```

Comment:
we are not adding major versions until we move to semantic import versioning. I would just say HEAD of master tracks Kubernetes master.